src/features/pmicControl/npm/npm1300/tests/commandCallbacks.test.ts

```diff
@@ -142,6 +143,83 @@ describe('PMIC 1300 - Command callbacks', () => {
         expect(mockOnChargerUpdate).nthCalledWith(1, { iTerm: '20%' });
     });
 
+    test.each(['get', 'set 1340'])('npmx charger iBatLim %p', append => {
+        const command = `npmx charger discharging_current ${append}`;
+        const callback =
+            eventHandlers.mockRegisterCommandCallbackHandler(command);
+
+        callback?.onSuccess('Value: 1340mA.', command);
+
+        expect(mockOnChargerUpdate).toBeCalledTimes(1);
+        expect(mockOnChargerUpdate).nthCalledWith(1, { iBatLim: 1340 });
+    });
+
+    test('npmx charger iBatLim apx result', () => {
+        const command = `npmx charger discharging_current set 271`;
+        const callback =
+            eventHandlers.mockRegisterCommandCallbackHandler(command);
+
+        callback?.onSuccess(
+            'Info: Requested value was 271 but reading will return 300 due to approximations.',
+            command
+        );
+
+        expect(mockOnChargerUpdate).toBeCalledTimes(1);
+        expect(mockOnChargerUpdate).nthCalledWith(1, { iBatLim: 271 });
+    });
+
+    test.each(['get', 'set 1000'])(
+        'npmx charger iBatLim closes to High should overwrite ans set it to high %p',
```

Comment:
```suggestion
        'npmx charger iBatLim closes to High should overwrite and set it to high %p.',
```
Why `High` capitalized? What does `closes to` mean?

Reply:
This test has been removed as feature request changed and this is no longe needed .... but what it meant was that give we only allowed 1340 and 271 if user set is to 500 we set it to 270 as it is closer to 271 than 1340